make-release.sh

```diff
@@ -108,8 +108,7 @@ fi
 if [[ $TRIGGER_RELEASE -eq 1 ]]; then	
   # push new branch to release branch to trigger CI build	
   git checkout "${BRANCH}"	
-  docker build -t "${QUAY_REPO}" -f apache.Dockerfile .	
-  docker push "${QUAY_REPO}"	
+  docker buildx build --platform linux/amd64,linux/s390x -t "${QUAY_REPO}" -f apache.Dockerfile  --push .
```

Comment:
please use a PLATFORMS file like in this PR https://github.com/eclipse/che-devfile-registry/pull/336

Reply:
also note that s390x fails w/ qemu and ubuntu, so this isn't a viable solution. Might have to look at travis + IBM services.